canonical-web-and-design/ubuntu.com
# Add adguard intel-nuc page

## Done

- Created the adguard intel-nuc install page
- Updated the raspberry-pi page to use the new includes

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/adguard/intel-nuc and http://0.0.0.0:8001/appliance/adguard/raspberry-pi
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the pages work as expected

## Issue / Card

Fixes #7497
